src/plugins/discover/public/application/components/sidebar/lib/field_calculator.js

```diff
@@ -20,9 +20,9 @@
 import _ from 'lodash';
 import { i18n } from '@kbn/i18n';
 
-function getFieldValues(hits, field) {
+function getFieldValues(hits, field, indexPattern) {
   const name = field.name;
-  const flattenHit = field.indexPattern.flattenHit;
+  const flattenHit = indexPattern.flattenHit;
```

Comment:
this fixes the exception that causes the field list to break on unknown fields 👍 
#75025
But still there's no name displayed after the refactoring introduced in #73180

Reply:
@mattkime you did! 🥳 thx